ethereum/blockcreation/src/test/java/org/hyperledger/besu/ethereum/blockcreation/AbstractBlockTransactionSelectorTest.java

```diff
@@ -1539,7 +1538,7 @@ private void internalBlockSelectionTimeoutSimulationInvalidTxs(
         .containsExactlyInAnyOrderEntriesOf(expectedEntries);
 
     assertThat(transactionPool.getTransactionByHash(lateTx.getHash()).isEmpty())
-        .isEqualTo(isLongProcessingTxDropped ? true : false);
+        .isEqualTo(isLongProcessingTxDropped);
```

Comment:
Don't understand why `.isEqualTo(isLongProcessingTxDropped ? true : false)` was changed to `.isEqualTo(isLongProcessingTxDropped)`.

Reply:
that was just cleaning up - it's equivalent - if `x=true ? true : false` is the same as `if x`